### PR TITLE
Update article

### PR DESCRIPTION
**Pull request by andyOlarte514**
## Summary
- Update Article

### Android
image
image

### iOS
image
image

## Trello Card
- https://trello.com/c/VyNKUFIG/8-update-article

### CODE REVIEW

File: src/app/screens/DetailArticle/index.tsx [hunk 1]
```diff
@@ -29,8 +32,40 @@ function DetailArticle({ route }: Props) {
...
+    () =>
+      navigation?.navigate(Routes.EditArticle, {
+        article: { ...route?.params?.article, isEditArticle: true }
+      }),
+    [navigation, route]
+  );
+
   return (
     <View style={styles.container}>
+      <View style={styles.containerActions}>
```
Comment (EstebanIndiveri) [inline src/app/screens/DetailArticle/index.tsx:47]: They can't be visible when user isn't login, can you use?
const currentUser = useSelector((state: State) => state.auth.currentUser);
{currentUser &&( ....)}

File: src/app/screens/DetailArticle/index.tsx [hunk 2]
```diff
@@ -8,13 +8,16 @@ import icAddActive from '@assets/TabBar/icAddpostActive.png';
...
+import icEdit from '@assets/icons/icEdit.png';
+import Routes from '@constants/routes';
 
 import styles from './styles';
 import testIds from './testIds';
 
 interface Props extends Navigation {}
 
-function DetailArticle({ route }: Props) {
+function DetailArticle({ route, navigation }: Props) {
```
Comment (EstebanIndiveri) [inline src/app/screens/DetailArticle/index.tsx:20]: what do you think about this :

 interface Props {
  route: {
    params: {
      article: Article;
    };
  };
}
and navigation from: import useNavigation from '@components/AppNavigator/helper'; 
if you don't like ignore this

**Review by EstebanIndiveri — CHANGES_REQUESTED** [review]
great! I have only 2 comments

**Review by EstebanIndiveri — APPROVED** [review]
good Job !!

File: src/app/screens/DetailArticle/index.tsx [hunk 3]
```diff
@@ -1,18 +1,30 @@
...
 import icDefaultArticleImage from '@assets/icons/icDefaultArticleImage.jpg';
+import icDelete from '@assets/icons/icDelete.png';
+import icEdit from '@assets/icons/icEdit.png';
+import Routes from '@constants/routes';
 
 import styles from './styles';
 import testIds from './testIds';
 
-interface Props extends Navigation {}
+export interface Props {
```
Comment (santiagorobra) [inline src/app/screens/DetailArticle/index.tsx:21]: I think this interface is going to be repeated a lot, could you make a general one?

route: {
    params: {
      
    };
  };

**Review by santiagorobra — APPROVED** [review]

**Pull request merged**